flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/handlers/CurrentJobsOverviewHandler.java

```diff
@@ -121,6 +125,26 @@ public String handleJsonRequest(Map<String, String> pathParams, Map<String, Stri
 		}
 	}
 
+	public static class CurrentJobsOverviewArchiver implements Archiver {
+
+		@Override
+		public ArchivedJson[] archiveJsonWithPath(AccessExecutionGraph graph) throws IOException {
```

Comment:
I don't quite understand, we are using it? The archiver follows the code path of the handler in case ```includeRunningJobs``` and ```includeFinishedJobs``` are both true, since this is the only path that the WebFrontend actually calls.

I don't remember what in the end cause the job status to not appear, it is fixed now though.